spring-integration-test-support/src/main/java/org/springframework/integration/test/condition/LogLevels.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2017-2019 the original author or authors.
```

Comment:
I don't think `2017` should be here, even if some logic has existed before in other classes.
Also I think we need to mention how do they combine together (if any) when we have a class and method level annotations.

Would be also great to mention this new feature in Docs!